lib/core/rancher/step.sh

```diff
@@ -167,7 +169,7 @@ function ih::setup::core.rancher::install() {
       elif ! grep -q "<string>vz</string>" "$PLIST_DST"; then
         ih::log::debug "Updating PLIST to use 'vz' for Virtualization for M3 Macs."
         sudo sed -i '' 's/<string>qemu<\/string>/<string>vz<\/string>/g' "$PLIST_DST"
-        $HOME/.rd/bin/rdctl set --experimental.virtual-machine.mount.type virtiofs
```

Comment:
I think I was thinking I had to change it from 9p and that’s way harder because it matches with several lines.

But this seems to work since there's just 1 match in the plist file for `reverse-sshfs`.

I'm trying to avoid having to maintain multiple copies of the plist file. Yeah, I know this is sorta hacky.

Reply:
At first glance it looks like it's set to 9p, but it's really set to use reverse-sshfs: https://github.com/ConsultingMD/homebrew-ih-public/blob/master/lib/core/rancher/io.rancherdesktop.profile.defaults.plist#L65-L79